templates/base.html

```diff
@@ -0,0 +1,98 @@
+{% load static %}
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+
+    <!-- Boostrap basic imports -->
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+          integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+
+    <!-- Customs -->
+    <title>{% block title %} {% endblock %}</title>
+    <link href="{% static 'main/css/style.css' %}" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.5.0/font/bootstrap-icons.css">
+        <link rel="shortcut icon" type="image/png" href="{% static 'favicon.ico' %}"/>
+    {% block styles %} {% endblock %}
+</head>
+
+<body>
+<div class="container">
+    <header class="p-3 bg-dark text-white">
+        <div class="container">
+            <div class="d-flex flex-wrap align-items-center justify-content-center justify-content-lg-start">
+                <a href="/" class="d-flex align-items-center mb-2 mb-lg-0 text-white text-decoration-none">
+                    <img src="{% static "main/img/logo.png" %}" alt="logo" width="40" height="45">
+                </a>
+                <ul class="nav col-12 col-lg-auto me-lg-auto mb-2 justify-content-center mb-md-0">
+                    <li><a href="{% url 'cats:index' %}" class="nav-link px-2 text-secondary">Home
+                    </a></li>
+                    <li><a href="{% url 'cats:species' %}" class="nav-link px-2 text-white">Species
+                    </a></li>
+                    <li><a href="{% url 'cats:rentals_history' %}" class="nav-link px-2 text-white">Rentals
+                    </a></li>
+                    <li><a href="{% url 'cats:about' %}" class="nav-link px-2 text-white">About
+                    </a></li>
+                </ul>
+
+                {#                <form class="col-12 col-lg-auto mb-3 mb-lg-0 me-lg-3">#}
+                {#                    <input type="search" class="form-control form-control-dark" placeholder="Search..."#}
+                {#                           aria-label="Search" control-id="ControlID-1">#}
+                {#                </form>#}
```

Comment:
Please do not commit commented code if it is not `TODO` with some description about it